orc8r/cloud/deploy/terraform/orc8r-helm-aws/variables.tf

```diff
@@ -203,7 +203,7 @@ variable "wifi_orc8r_chart_version" {
 variable "orc8r_tag" {
   description = "Image tag for Orchestrator components."
   type        = string
-  default     = "1.5.0"
+  default     = "latest"
```

Comment:
I don't think this is a good idea -- in fact, I think we should just remove the default altogether, and force users to include a value in their main.tf

Reply:
hmmm... I kind of disagree, most packet system gives you the latest stable version. That's what it would do and now that we switched to the new artifactory we have only versions starting from 1.5.0.
And there is this pattern in magma where if you want to deploy an older version, you would have to checkout the tag at this version. This aligns with that idea.
I could put 1.6.0 to align with what was there before, but removing it here is kind of beyond the scope of the PR.

Reply:
Makes sense. Okay yeah let's just go with 1.6.0 I think for all these